alert-policies/activemq/MessageDequeueRate.yml

```diff
@@ -0,0 +1,27 @@
+name: Message Dequeue Rate
+
+description: |+
+  This alert is triggered when the Messages received is less than 1 during 5 minutes.
```

Comment:
OLD:
This alert is triggered when the Messages received is less than 1 during 5 minutes.
Update needed to:
This alert is triggered when the messages received are less than 1 during 5 minutes.

Reply:
Hi @pkosanam,
Updated grammar mistake changes.